modules/imaging_browser/php/NDB_Menu_Filter_imaging_browser.class.inc

```diff
@@ -273,7 +294,9 @@ class NDB_Menu_Filter_Imaging_Browser extends NDB_Menu_Filter
         $user =& User::singleton();
 
```

Comment:
@gluneau
done, thanks for the catch!
I also caught another set of php warnings in my error_log as a result of your comment, and I solved them in the changes on lines 69 and 74